src/DungeonsDragonsTracker/DungeonsDragonsTracker.tsx

```diff
@@ -10,8 +11,20 @@ export const DungeonsDragonsTracker = () => {
 
 
     return <div className={cx('trackerMain')}>
-        <div className={cx('headerBar')}>Header</div>
-        <div className={cx('mainBody')}></div>
+        <div className={cx('headerBar')}><div>Menu</div><div>character/player info </div><div> save button</div></div>
```

Comment:
This is the grouping I'm thinking for the project. Can be re-grouped as needed. Don't have any styling in this yet but imagine we will add in as we finish sections.